.github/workflows/build-llvm.yml

```
@@ -0,0 +1,204 @@
+            --manifest "${MANIFEST}" \
+            --install-dir ".llvm/build-install/lib" \
+            --build-dir "build" \
+            --gh-run-id "${{ github.run_id }}" \
+            --gh-artifact "llvm-pruned-libs-${{ matrix.os }}" \
+            --gh-download-dir "artifacts" \
+            --max-attempts 60 \
+            --sleep-seconds 60
+
+      - name: Package LLVM install directory
+        shell: bash
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2193:warning:12:28: The arguments to this comparison can never be equal. Make sure your syntax is correct

(shellcheck)